charts/merlin/templates/_helpers.tpl

```diff
@@ -322,7 +337,7 @@ MlflowConfig:
 {{ printf "%s%s:%s" (ternary (printf "%s/" $.Values.deployment.image.registry) "" (ne $.Values.deployment.image.registry "")) $.Values.deployment.image.repository $.Values.deployment.image.tag }}
 {{- else -}}
 {{- with index . 1 }}
-{{- $tag :=  ternary $.Values.deployment.image.tag (substr 1 (len $rendered.releasedVersion) $rendered.releasedVersion) (ne $.Values.deployment.image.tag "") -}}
+{{- $tag :=  include "merlin.version" . -}}
```

Comment:
iinw, this is a corner case, but at the last else while rendering this we are defaulting to ChartAppVersion. which will be with the prefix `v`. I think that needs to be removed like we do with rendered version.